Exercises/4-count-types.js

```diff
@@ -1,5 +1,13 @@
 'use strict';
 
-const countTypesInArray = null;
+const countTypesInArray = array => {
```

Comment:
@tshemsedinov how about?
```js
let arr = ['string', NaN, 4, () => {}, { a: 1}];
[...new Set(arr.map(el => typeof el))].length;
```

Reply:
Don't invent slow and complex solutions. Use `for..of`, not `Set`, `map` and spread.